VAMobile/src/api/directDeposit/updateBankInfo.tsx

```diff
@@ -27,10 +27,13 @@ export const useUpdateBankInfo = () => {
 
   return useMutation({
     mutationFn: updateBankInfo,
-    onSuccess: async () => {
+    onSuccess: async (data, newAccountData) => {
       logAnalyticsEvent(Events.vama_update_dir_dep())
       setAnalyticsUserProperty(UserAnalytics.vama_uses_profile())
-      queryClient.invalidateQueries({ queryKey: directDepositKeys.directDeposit })
+      const queryData = queryClient.getQueryData(directDepositKeys.directDeposit) as DirectDepositData
+      newAccountData.financialInstitutionName = 'Bank'
```

Comment:
Does this mean we're always setting financialInstitutionName to "Bank"? Not sure how this field is normally used, or if it could contain something else that we're overwriting here

Reply:
yeah in the original slice we set the data manually to bank when we sent it to the server, so I kept that in line here